python/lsst/pex/logging/__init__.py

```diff
@@ -20,5 +20,6 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
+import loggingLib
```

Comment:
are you sure this shouldn't be `from .loggingLib import *`? And why are there two imports here for `loggingLib`?